README.md

```diff
@@ -1,4 +1,161 @@
 Kivy SDK Packager
 =================
 
-This repository contains scripts for Kivy SDK generation on Windows, OS X and Linux.
+Kivy Software Development Kit (Kivy SDK) is an installable package of all the 
+basic dependencies needed to run [Kivy framework](https://kivy.org) apps.
+
+Installing them as a single package may save time.
+
+The Kivy SDK Packager (kivy-sdk-packager) repository contains
+the definitions of the SDK for each of three different platforms, and the tools 
+to build and distribute them.
```

Comment:
Not properly, `kivy-sdk-packager` has multiple purposes (and I wonder if would be better to split it into multiple repos in the future).

- Windows:
    - We have scripts to build dependencies to run Kivy (Such as `sdl2`, `glew`and `angle`)
 - Linux (debian):
    - `kivy-sdk-packager` hosts the files needed during the execution of the build on Ubuntu PPA
 - macOS:
     - `kivy-sdk-packager` hosts the scripts needed to package a `.dmg` or `.app` (can be the `Kivy.app` we distribute when a new Kivy release is produced, or can be used by our users to package their Kivy-based app for macOS)